### DISCUSSION TITLE
Migrating from Google photos

### DISCUSSION THREAD

**Discussion by nautatva** [photoprism/photoprism#1894] [Q&A] [answered]
With not so recent policy changes, I have decided to move to using photoprism as my primary photo manager. I have used Google takeout to export all the data from Google photos. How do I import the data to photoprism (Google gives photo and a extra JSON with every photo on takeout)

**Comment by lastzero** [comment, marked as answer]
https://docs.photoprism.app/user-guide/use-cases/google/
